rocky/rocky/templates/tasks/partials/task_filter.html

```diff
@@ -1,20 +1,46 @@
 {% load i18n %}
+<div class="filter">
+  <div>
+      <p>{% translate 'Filter' %}</p>
+      <button aria-expanded="false"
+              data-hide-filters-label="{% translate 'Hide filters' %}">
+        {% translate 'Show filters' %}
+      </button>
+  </div>
 
-{% if task_filter %}
-  <form method="get">
-    {% csrf_token %}
-    <label for="selected-tasks">{% translate 'Number of tasks showed' %}</label>
-    <select id="selected-tasks" name="selected-tasks">
-      {% for task in task_filter %}
-        <option {% if task == show_tasks %}selected{% endif %} value={{ task }}>
-          {% if task == "*" %}
-            {% translate 'Show all' %}
-          {% else %}
-            {{ task }}
+  <form action="" method="get" novalidate method="get">
+    <fieldset>
+        <label for="scan_history_from">{% translate 'From' %}</label>
```

Comment:
We need to make sure we let the user know in which timezone (or not) we are asking this. Eg, it needs to correspond to what the backend is using.